### DISCUSSION TITLE
Jump from Failed Tests to same request in Total Requests?

### DISCUSSION THREAD

**Discussion by DavidAccola** [DannyDainton/newman-reporter-htmlextra#405] [General]
When viewing, for example, Failed Tests, is there any way to jump to see that failed request in context of Total Requests, or to otherwise get an idea of the request that preceded it?

Use case:
I am working on a battery of tests where the request that came before gives context to the request that comes next. Example:

Request 1: Class
√ Does the class exist?
√ Does the class have an ID that associates it to a teacher record?

Request 2: Teacher
X For the ID that class lists, does a teacher record for that ID actually exist?

When scrolling through all requests, you understand what the failed requests mean because you understand what came before. The issue comes when you jump over to Failed Tests. You see failures indicating the teacher associated with "that class" does not exist, but can't tell which class "that class" is out of context.

**Comment by DannyDainton** [comment]
To be honest, I can only do so much with the data coming from Newman. The Failed Tests tab was something that I originally did as it was easy to get those in an array from the Newman object and present them in that way, similar to what you would see for the skipped tests. Without the context, they are pretty meaningless. 

The failures within individual requests were something I wanted to add after seeing that there was no link between the two. Without doing some very hacky coding and spending a bunch of time, it's going to be difficult for me to invest that effort into doing that. :( 

The beauty of open-source software is that if you wanted to get it a go, feel free to send me a PR with the changes to review. ❤️

**Reply by DavidAccola** [reply]
That's fair! I suspected there was maybe no good way to do it.

I've considered going the opposite direction of modifying my template to remove Failed Requests so my users can rely entirely on Total Requests > Show Failed Iterations in context.

The reason I haven't done that is because some types of errors show only in Failed Tests, ex. ETIMEDOUT, ENOTFOUND, ECONNRESET. Is there any hope of an option for showing these errors in context in with Total Requests? I wasn't sure if that'd be easier than the first suggestion.
